nautobot_floor_plan/forms.py

```diff
@@ -181,7 +181,7 @@ class Meta:
         """Meta attributes."""
 
         model = models.FloorPlanTile
-        fields = "__all__"
+        exclude = ["allocation_type", "on_group_tile"] # pylint: disable=modelform-uses-exclude
```

Comment:
I think we our standard may still be to have both __all__ and exclude... I could be wrong, but pylint-nautobot may squawk once turned on later.